doc/private_net.rst

```diff
@@ -0,0 +1,101 @@
+Creating a private network and deploying a contract with Remix
+===============
+
+This guide describes creating a private network with a single node and connecting `Remix <https://remix.ethereum.org/>`__ to it through HTTP RPC interface.
+
+Initializing aleth in the Command-Line Interface
+------
+
+**1.** Create a new aleth-account.
+::
+  aleth/aleth account new
+  
+You are then asked to enter and confirm a password for this account.
```

Comment:
Can be rephrased as: "Enter and confirm a new password for this account when prompted."